pkgs/games/eduke32/default.nix

```diff
@@ -22,11 +23,12 @@ in stdenv.mkDerivation {
   inherit version;
 
   src = fetchurl {
-    url = "http://dukeworld.duke4.net/eduke32/synthesis/latest/eduke32_src_${version}-${rev}.tar.xz";
-    sha256 = "09a7l23i6sygicc82w1in9hjw0jvivlf7q0vw8kcx9j98lm23mkn";
+    url = "http://dukeworld.duke4.net/eduke32/synthesis/latest/eduke32_src_20200907-9257-93f62bbad.tar.xz"; # Good
```

Comment:
Shouldn't this be interpolated as before?

Reply:
```nix
 url = "http://dukeworld.duke4.net/eduke32/synthesis/latest/eduke32_src_${version}-${rev}-${revExtra}.tar.xz";
```

Reply:
You're right, I forgot to restore this. I'll fix this in a separate PR.

Reply:
Done in #104936.